chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -494,3 +494,12 @@ filter.notPresent=No filter object was present to be saved.
 filter.empty=Filter cannot be empty.
 filter.noUser=The filter does not have an associated user.
 filter.noFilterType=The filter does not have an associated filter type.
+
+complaints.create.statusOpen=When creating a new complaint, the Complaint Status must be open.
+complaints.create.statusOpen.error=There was an error in validator: ComplaintStatusOpen
```

Comment:
These error messages are supposed to be user friendly to non-developers. These messages can (maybe should?) go through the product owner for verification too. At least, that was one of the initial reasons for this file at all